### PR TITLE
Add newline after each json report object

### PR DESCRIPTION
**Pull request by mrhillsman**
Signed-off-by: Melvin Hillsman <mhillsma@redhat.com>

### CODE REVIEW

**Comment by mrhillsman** [comment]
Resolves #314

**Review by skattoju — APPROVED** [review]
/lgtm

**Comment by yashoza19** [comment]
Fixes #314

File: internal/report/report.go [hunk 1]
```diff
@@ -102,4 +102,4 @@ func OperandInstallTextReport(w io.Writer, data TemplateData) error {
 
 func OperandInstallJsonReport(w io.Writer, data TemplateData) error {
 	return processTemplate(w, operandJsonReportTemplate, data)
-}
+}
```
Comment (bcrochet) [inline internal/report/report.go:105]: Don't remove the newline at the end.

**Review by bcrochet — CHANGES_REQUESTED** [review]
go.mod and go.sum probably don't need to change here.

**Pull request closed without merging**